chat.js

```diff
@@ -1190,7 +1190,7 @@ Chat.getDataPokemonHTML = function (template, gen = 7) {
 Chat.getDataMoveHTML = function (move) {
 	if (typeof move === 'string') move = Object.assign({}, Dex.getMove(move));
 	let buf = `<ul class="utilichart"><li class="result">`;
-	buf += `<a data-entry="move|${move.name}"><span class="col movenamecol">${move.name}</span> `;
+	buf += `<a data-entry="move|${move.name}"><span class="col movenamecol"><a href="https://pokemonshowdown.com/dex/moves/${move.name}${move.name}</a></span> `;
```

Comment:
Still wrong...

Reply:
Here's what the link part should look like:

`<a href="https://pokemonshowdown.com/dex/moves/${move.id}">${move.name}</a>`

Reply:
Like, links in the abstract look like this:

`<a href="URL">LINK TEXT</a>`

Reply:
Oh, duh. I guess I'm still really bad at html.